projects/1.20/assets/goldelight/goldelight/lang/zh_cn.json

```
@@ -0,0 +1,313 @@
+  "block.goldelight.golden_soil": "金泥土",
+  "block.goldelight.golden_soil_farmland": "金耕地",
+  "block.goldelight.golden_stairs": "金楼梯",
+  "block.goldelight.golden_stem": "金菌柄",
+  "block.goldelight.golden_sugar_cane": "金甘蔗",
+  "block.goldelight.golden_tomato_crate": "箱装金番茄",
+  "block.goldelight.golden_tomatoes": "金番茄",
+  "block.goldelight.golden_trapdoor": "金活板门",
+  "block.goldelight.golden_wall_hanging_sign": "墙上的悬挂式金告示牌",
+  "block.goldelight.golden_wall_sign": "墙上的金告示牌",
+  "block.goldelight.golden_wart": "金下界疣",
+  "block.goldelight.golden_wart_block": "金下界疣块",
+  "block.goldelight.golden_wheat": "金小麦",
+  "block.goldelight.gray_candle_golden_cake": "插上灰色蜡烛的金蛋糕",
+  "block.goldelight.green_candle_golden_cake": "插上绿色蜡烛的金蛋糕",
+  "block.goldelight.groomlight": "金菌光体",
+  "block.goldelight.light_blue_candle_golden_cake": "插上淡蓝色蜡烛的金蛋糕",
+  "block.goldelight.light_gray_candle_golden_cake": "插上淡灰色蜡烛的金蛋糕",
+  "block.goldelight.lime_candle_golden_cake": "插上黄绿色蜡烛的金蛋糕",
+  "block.goldelight.magenta_candle_golden_cake": "插上品红色蜡烛的金蛋糕",
+  "block.goldelight.nether_brick_stove": "下界砖炉灶[WIP]",
+  "block.goldelight.orange_candle_golden_cake": "插上橙色蜡烛的金蛋糕",
+  "block.goldelight.pink_candle_golden_cake": "插上粉红色蜡烛的金蛋糕",
+  "block.goldelight.potted_golden_cactus": "金仙人掌盆栽",
+  "block.goldelight.potted_golden_fungus": "金黄菌盆栽",
+  "block.goldelight.potted_golden_mushroom": "金蘑菇盆栽",
+  "block.goldelight.purple_candle_golden_cake": "插上紫色蜡烛的金蛋糕",
+  "block.goldelight.red_candle_golden_cake": "插上红色蜡烛的金蛋糕",
+  "block.goldelight.stripped_golden_hyphae": "去皮金菌核",
+  "block.goldelight.stripped_golden_stem": "去皮金菌柄",
+  "block.goldelight.stuffed_golden_pumpkin_block": "金填馅南瓜",
+  "block.goldelight.sweet_golden_berry_cheesecake": "金甜浆果芝士派",
+  "block.goldelight.white_candle_golden_cake": "插上白色蜡烛的金蛋糕",
+  "block.goldelight.yellow_candle_golden_cake": "插上黄色蜡烛的金蛋糕",
+  "effect.goldelight.immunity": "免疫",
+  "effect.goldelight.immunity.desc": "清除负面效果",
+  "effect.goldelight.immunity.description": "清除负面效果",
+  "effect.goldelight.mildness": "温暖",
+  "effect.goldelight.mildness.desc": "防止被冻结",
+  "effect.goldelight.mildness.description": "防止被冻结",
+  "enchantment.goldelight.midas_touch": "迈达斯之触",
+  "enchantment.goldelight.midas_touch.desc": "一些水滴会变成黄金。",
+  "entity.goldelight.golden_bee": "金蜜蜂",
+  "entity.goldelight.golden_chicken": "金鸡",
+  "entity.goldelight.golden_cow": "金牛",
+  "entity.goldelight.golden_egg": "金鸡蛋",
+  "entity.goldelight.rotten_golden_tomato": "烂金番茄",
+  "farmersdelight.tooltip.golden_dog_food.when_feeding": "喂给已驯服的狗时：",
+  "farmersdelight.tooltip.golden_horse_feed.when_feeding": "喂给已驯服的马时：",
+  "farmersdelight.tooltip.golden_hot_cocoa": "保护你免受冻结",
+  "farmersdelight.tooltip.golden_melon_juice": "轻微瞬间治疗",
+  "farmersdelight.tooltip.golden_milk_bottle": "清除1个效果",
+  "goldelight.advancement.avarice": "贪婪的化身",
+  "goldelight.advancement.avarice.desc": "吃下一顿黄金套餐！",
+  "goldelight.advancement.drink_golden_hot_cocoa": "无论如何，寒冷从未困扰过我",

```

```suggestion
  "goldelight.advancement.drink_golden_hot_cocoa": "寒意再也不会困扰我了",
```
出典《冰雪奇缘》片中曲《Let It Go》
说实话这句歌词可能还是英文更广为人知？（）
采用网易云翻译

---

事实上，网易云一如既往地在翻译上拉了跨
曲子的措辞偏日常，所以实际上最好就用日常措辞翻译，比如
“反正我本来也不怕冷”

---

但是Chui这里的译文实际上契合了模组本身的属性，所以还是译者自己看着办吧